images.CI/linux-and-win/lint_ubuntu.ps1

```diff
@@ -0,0 +1,34 @@
+param(
+    [Parameter(Mandatory)] [string] $path,
+    [Parameter(Mandatory)] [string] $pattern
+)
+
+$ErrorActionPreference = "Stop"
+
+function Validate-Scripts {
+    Param (
+        [Parameter(Mandatory=$true)]
+        [string[]]$Path,
+        [Parameter(Mandatory=$true)]
+        [string]$Pattern
+    )
+     $WrongScript = New-Object System.Collections.Generic.List[System.Object]
+     Get-ChildItem $path | ForEach-Object {
+       if (Select-String -Path $($_.FullName) -Pattern $Pattern -Quiet) {
+          Write-Host "Pattern '$pattern' found in '$($_.FullName)'"
+       }
+       else {
+          Write-Host "Pattern '$pattern' not found in '$($_.FullName)'"
+          $WrongScript += $($_.FullName)
+       }
+    }
+    return $WrongScript
+}
+
+$FailedScripts = Validate-Scripts -Path $path -Pattern "#/bin/bash -e"
```

Comment:
I suggest to implement the single script for Linux and MacOS.
Actually, you can just invoke this function twice:
```
Validate-Scripts -Path “images/macos” -Pattern “<pattern1>”
Validate-Scripts -Path “images/linux” -Pattern “<pattern2>”
```

Reply:
I have moved script directly to images.CI folder